portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1437,6 +1455,49 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 	'getPatientHeatmapData': function(genetic_profile_id, genes) {
 	    return getHeatmapDataCached(this, genetic_profile_id, genes, 'patient');
 	},
+	'getClusteringOrder': function (case_ui_map, track_uid_map, heatmapData, case_ids) {
```

Comment:
Since you're returning a promise, I'd typically fetch `case_ui_map` again from `this.getCaseUIDMap` instead of passing it in.